### ISSUE TITLE
twisted/web/test/test_http.py uses backticks

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2382]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2382|
|Type|defect|
|Created|2007-01-10 20:38:40Z|

Backticks are forbidden by the coding standard and also silly.

Attachments:

* [test_http_2382.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/620/620a913cf4d7ac817a545cbf7b999cf9812abf10/99c85c528ae37d360b43cd2a679b0de6086424f8.diff) (2896 bytes) - added by therve on 2007-01-12 10:39:08Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__2382 2382
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1168461520000000 1168461520000000
changetime__1169240280000000 1169240280000000
version__None None
owner__ 
cc__therve
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve removed owner|
|-|-|

I took the opportunity to remove the trailing spaces and add a doctest.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve commented|
|-|-|

I meant docstring.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @therve|
|-|-|

Looks good!  Please apply.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [19414]) Author: therve
Reviewer: exarkun
Fixes [#2382](https://github.com/twisted/twisted/issues/2382)

Remove backticks from twisted.web.test.test_http, and add a docstring.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
